crates/svm-ffi/src/env.rs

```diff
@@ -0,0 +1,57 @@
+use std::ffi::c_void;
+
+use crate::svm_wasm_types_t;
+
+use svm_types::WasmType;
+
+#[allow(non_camel_case_types)]
+#[derive(Clone)]
+#[repr(C)]
+pub struct svm_env_t {
+    pub inner_env: *const c_void,
+
+    pub host_env: *const c_void,
+
+    pub returns: svm_wasm_types_t,
```

Comment:
What's the purpose of this field? Will it ever need to be used by the extern client?
I see that `svm` use it in `inner_callback` to allocate the `results` buffer, but I don't think it's necessary - why not just read it via closure, instead of doing set/get via `env`?